setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
It's actually Paul Wolf's project - I just thought I'd suggest this very minor tweak... BTW, I amended my commit with your for-loop suggestion.